float32 roll
float32 pitch
float32 pressure
float32 temperature
